freqtrade/rpc/rpc.py

```diff
@@ -1097,19 +1097,23 @@ def _rpc_delete(self, trade_id: int) -> dict[str, str | int]:
                 "cancel_order_count": c_count,
             }
 
-    def _rpc_list_custom_data(self, trade_id: int, key: str | None) -> list[dict[str, Any]]:
-        # Query for trade
-        trade = Trade.get_trades(trade_filter=[Trade.id == trade_id]).first()
-        if trade is None:
+    def _rpc_list_custom_data(self, trade_id: int) -> list[dict[str, Any]]:
```

Comment:
well won't this change (removal of the key argument) break how the telegram integration works? (written before CI has ran)